source/aggregation-pipeline-builder.txt

```diff
@@ -205,8 +205,12 @@ your chosen view mode:
             On the :guilabel:`Stage Wizard` card, select a stage use 
             case for the first stage of the pipeline. You can click the stage 
             card to add it to the end of your pipeline or drag it to your 
-            preferred position. |compass-short| populates the stage card with a 
-            form that corresponds with the selected aggregation pipeline stage.
+            preferred position. 
+            
+            You can also filter the use cases by searching for the associated 
```

Comment:
[ordering]

Would it make sense to move the first sentence of this paragraph to the preceding paragraph? It might present a slightly more logical for the flow to be along the lines of:

1. Search for a use case / stage
2. Add the stage
3. Compass populates the stage card

Again, I'm not actually looking at the tool, so my understanding might be off here.

Reply:
I'm thinking I might just add an **optional** step before this one that says search for your stage use case. Otherwise, I feel like it could clutter this step and give procedures for more than one action.